index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css" />
+  <title>Explore the Tour: Warner Bros Studio Tour</title>
+</head>
+<body>
+  <header id="landing">
+    <nav>
+      <div class="logo">
+        <a href="./index.html">
+          <img src="./media/Warner-Bros-Logo-1.png" alt="navigation logo image" class="logo-img">
+        </a>
+      </div>
+      <div class="nav-centre">
+        <ul class="nav-links">
+          <li class="nav-link"><a href="./about.html">About</a></li>
+          <li class="nav-link"><a href="#experience">Experience</a></li>
+          <li class="nav-link"><a href="#featured">What's On</a></li>
+          <li class="nav-link"><a href="#awards">Awards</a></li>
+          <li class="nav-link"><a href="#contact-us">Contact us</a></li>
+        </ul>
+      </div>
+      <div class="nav-end">
+        <button id="tickets-btn">Tickets</button>
+      </div>
+      <div class="menu-button-wrap">
+        <div class="menu-line"></div>
+      </div>
+    </nav>
+    <section class="heading">
+      <p class="supporting-heading">Discover the Filmmaking Magic</p>
+      <h1 class="main-heading">Explore the wonders of the Wizarding World</h1>
+      <div class="description">
+        <p class="description-text">The Studio Tour offers visitors the unique opportunity to explore two soundstages and a backlot filled with original sets, animatronic creatures and breathtaking special effects.</p>
+      </div>
+      <div class="booking-section">
+        <button id="book-now-btn">Book Now</button>
+        <p class="booking-description">MUST BE BOOKED IN ADVANCE</p>
+      </div>
+    </section>
+  </header>
+  <main>
+    <section id="experience" class="experience-section">
+      <h3 class="experience-heading">The Experience</h3>
+      <div id="experience-heading-under"></div>
+      <ul class="experiences-list">
+        <li id="experience-1" class="experience">
+          <div class="experience-image-1"></div>
+          <h4 class="experience-main">Sets</h4>
+          <p class="experience-info">Step into the Great Hall, Forbidden Forest and Diagon Alley</p>
```

Comment:
- [ ] Kindly implement the `hover` functionality of your cards to match the original design

![image](https://user-images.githubusercontent.com/98453979/204053705-3d52d5d9-e502-4c93-9a82-3e2b6f8dabfb.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/204053563-a0073388-0303-461c-87c8-feaeb9b75fc5.png)